rasa/shared/core/training_data/story_reader/yaml_story_reader.py

```diff
@@ -139,6 +135,9 @@ def read_from_parsed_yaml(
     def is_stories_file(cls, file_path: Text) -> bool:
         """Check if file contains Core training data or rule data in YAML format.
 
+        Will throw exceptions if the file seems to be a yaml file (extension) but
```

Comment:
The google docstring define a `Raises:` syntax https://google.github.io/styleguide/pyguide.html#244-decision

Reply:
ah nice, wasn't aware of that :+1: